gcc/rust/rust-session-manager.cc

```diff
@@ -53,6 +53,7 @@ const char *kLexDumpFile = "gccrs.lex.dump";
 const char *kASTDumpFile = "gccrs.ast.dump";
 const char *kASTExpandedDumpFile = "gccrs.ast-expanded.dump";
 const char *kHIRDumpFile = "gccrs.hir.dump";
+const char *kHIRPrettyDumpFile = "gccrs.hir.pretty.dump";
```

Comment:
```suggestion
const char *kHIRPrettyDumpFile = "gccrs.hir-pretty.dump";
```
nit but this way it stays inline with the other file options :)